### ISSUE TITLE
exciton-phonon coupling

### ISSUE THREAD

**Issue by similt** [pyscf/pyscf#2304]
Would it be possible to calculate exciton-phonon coupling in pyscf?

**Comment by sunqm** [comment]
This is not available in the current version. If you can instead map your problem to a molecule, the electron-phonon coupling is available in the `pyscf.eph` module.

**Comment by similt** [comment]
Yes. I am looking for exciton-phonon coupling in molecule. Can I approximate g_pq (eqn. 23) from  J. Chem. Phys. 153,  224112 (2020) with p=q=LUMO as excition-phonon coupling?

**Comment by yangcal** [comment]
I'm not an expert on exciton physics but I don't think one can approximate `g_pq` as exciton-phonon coupling just using p=q=LUMO. Exciton refers to a bound state formed by both electron and hole and even in the mean field picture, that should include both HOMO and LUMO. One may take the off-diagonal block where p=HOMO and q=LUMO to get an insight but that would still be a very crude approximation as exciton is naively treated as a pair of single particles, neglecting many-body and dynamic effects.

**Comment by tberkel** [comment]
The exciton-phonon coupling is commonly calculated by contracting the electron-phonon and hole-phonon couplings with the exciton wavefunctions, e.g., see Eq (47) in https://doi.org/10.1103/PhysRevB.105.085111

**Comment by verena-neufeld** [comment]
Hi @similt , thanks for asking. Has the question been resolved and can this issue be closed?
